src/seedu/addressbook/commands/Hello.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.commands;
+
+
+public class Hello extends Command {
+
+    public static final String COMMAND_WORD = "hello";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": prints hello World!";
+
+
+
+
+    public CommandResult execute() {
```

Comment:
Missing header comment. All non-trivial methods (in this case your enhanced code) should have javadoc format header comments.